docs/adguard-for-android/solving-problems/low-level-settings.md

```diff
@@ -132,39 +136,39 @@ Here you can specify port ranges that should be filtered.
 
 #### Log removed HTML events
 
-If enabled, AdGuard records blocked HTML elements in the filtering log.
+If enabled, AdGuard records blocked HTML elements in *Recent activity*.
 
 #### Scriplet debugging
 
-If you need to activate debugging of scriptlets, enable this feature. Then there will be messages in the browser log that some scriplet rules have been applied.
+This options displays information about applied scriptlet rules to the browser log.
```

Comment:
If this is enabled, debugging in scriptlets will be activated, and the browser log will record when scriptlet rules are applied.